Makefile

```diff
@@ -0,0 +1,28 @@
+
+DOCKERFILE := Dockerfile
+REGISTRY := quay.io
+REPOSITORY := openshiftio/osa-api-server
```

Comment:
Is this repository already created on quay.io?

Reply:
I raised one PR : https://gitlab.cee.redhat.com/service/app-interface/merge_requests/3667.
Is this sufficient or anything else needs to be done?